src/auth/auth.dtos.ts

```diff
@@ -0,0 +1,16 @@
+export class DiscordUser {
+    discord_username: string;
+    discriminator: string;
+    _id: string;
+    avatar: string;
+    provider_id: string;
+    accessToken?: string;
+    refreshToken?: string;
+}
+
+export class OauthSession {
+    governatorId: string;
+    status: number;
+    // expires: string,
```

Comment:
expires?

Reply:
just a placeholder. We maybe want to pass info as to when the session expires. Lets test first how the current implementation works, maybe add it later if it makes sense.